### PR TITLE
Bouncycastle 1.52 (needed to fix severe performance problems with disk encryption)

### PR DESCRIPTION
**Pull request by toad**
Tested locally, auto-update works (i.e. the dependencies.properties is correct).

### CODE REVIEW

**Comment by toad** [comment]
IMHO this is higher priority and lower risk than #381

File: README.building.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Install junit 4, Apache ant and Hamcrest. For Debian Wheezy these are `junit4`,
 If sufficient packages are not available, put these jars into the `lib/` directory:
 
 * `junit4.jar` and `hamcrest-core.jar`: See the [download instructions](https://github.com/junit-team/junit/wiki/Download-and-Install)
-* `bcprov-jdk15on-151.jar`: Download Bouncy Castle 1.51: `wget --no-passive-ftp ftp://ftp.bouncycastle.org/pub/release1.51/bcprov-jdk15on-151.jar` or `wget http://www.bouncycastle.org/download/bcprov-jdk15on-151.jar`
+* `bcprov-jdk15on-152.jar`: Download Bouncy Castle 1.52: `wget --no-passive-ftp ftp://ftp.bouncycastle.org/pub/release1.51/bcprov-jdk15on-152.jar` or `wget http://www.bouncycastle.org/download/bcprov-jdk15on-152.jar`
```
Comment (nextgens) [inline README.building.md]: This is obviously wrong...

**Comment by Thynix** [comment]
Anything else or are people happy with this?

**Comment by nextgens** [comment]
It doesn't update build.properties nor the eclipse stuff but yes, no further objection to what's in so far.

**Pull request merged**